config/Dockerfiles/singlehost-test/package-test.sh

```diff
@@ -66,12 +66,14 @@ fi
 
 # This will introduce a problem with concurrency as it has no locks
 function clean_up {
-if [[ -z "${RSYNC_USER}" || -z "${RSYNC_SERVER}" || -z "${RSYNC_DIR}" || -z "${RSYNC_PASSWORD}"  || -z "${RSYNC_BRANCH}" ]]; then echo "Told to rsync but missing rsync env var(s)" ; exit 1 ; fi
-     RSYNC_LOCATION="${RSYNC_USER}@${RSYNC_SERVER}::${RSYNC_DIR}/${RSYNC_BRANCH}"
-     rm -rf tests/package
-     mkdir -p tests/package
-     cp ${TEST_ARTIFACTS}/* tests/package/
-     rsync --stats -arv tests ${RSYNC_LOCATION}/repo/${package}_repo/logs
+    RSYNC_LOCATION="${RSYNC_USER}@${RSYNC_SERVER}::${RSYNC_DIR}/${RSYNC_BRANCH}"
```

Comment:
looks like the RSYNC_LOCATION line should be deleted here..  You define it inside the if statement below...

Reply:
Thanks